docs/source/release/v6.4.0/framework.rst

```diff
@@ -51,6 +51,13 @@ Bugfixes
 
 .. amalgamate:: Framework/Fit_Functions/Bugfixes
 
+Kernel
+------
+
+Bugfixes
+########
+
+.. amalgamate:: Framework/Kernel/Bugfixes
```

Comment:
The amalgamate directives were removed from framework.rst on the release-next branch last week because the release notes are managed manually in the last couple of weeks before the release. Please can you rebase onto release-next and just leave the "Kernel" and "Bugfixes" headings in here? The separate 33896.rst file is fine.

Someone in the team here is manually merging in the issue.rst files in batches so your note will get merged at some point before the release.

Reply:
I modified the previous commit. I removed the changes made on `docs/source/release/v6.4.0/framework.rst` and only kept the `33896.rst` file.